win32ss/gdi/ntgdi/freetype.c

```diff
@@ -1365,11 +1361,9 @@ IntGdiLoadFontsFromMemory(PGDI_LOAD_FONT pLoadFont)
             Entry->NotEnum = (Characteristics & FR_NOT_ENUM);
             InsertTailList(&LoadedFontList, &Entry->ListEntry);
 
-            DPRINT("Font loaded: %s (%s)\n",
-                   Face->family_name ? Face->family_name : "<NULL>",
-                   Face->style_name ? Face->style_name : "<NULL>");
+            DPRINT("Font loaded: %s (%s), CharSet %u\n", Face->family_name, Face->style_name,
+                   FontGDI->CharSet);
```

Comment:
The next dprint (num_glyphs) can be made at the same time here instead.

Reply:
OK.